storages/backends/s3boto.py

```diff
@@ -494,3 +495,9 @@ def get_available_name(self, name, max_length=None):
             name = self._clean_name(name)
             return name
         return super(S3BotoStorage, self).get_available_name(name, max_length)
+
+    def update_headers(self, headers, name=None, content=None):
+        """
+        Method that can be used to adjust file parameters.
```

Comment:
should it be "Method that can be used to adjust file **headers**."?

Reply:
yeah, right, I'll change that